src/ansys_sphinx_theme/__init__.py

```diff
@@ -156,8 +156,11 @@ def setup_default_html_theme_options(app):
     # Place all switchers and icons at the end of the navigation bar
     if theme_options.get("switcher"):
         theme_options.setdefault(
-            "navbar_end", ["version-switcher", "theme-switcher", "navbar-icon-links"]
+            "navbar_end",
+            ["search-field", "version-switcher", "theme-switcher", "navbar-icon-links"],
```

Comment:
```suggestion
            ["search-button-field", "version-switcher", "theme-switcher", "navbar-icon-links"],
```

Reply:
I guess the difference is that `search-button-field` shows the keyboard shortcut?

Reply:
> search field.html is the pop-up search bar from pydata sphinx theme, that we avoid completely as per ux instruction, and search button field is the inbuild search for layout. We overwrite the https://github.com/ansys/ansys-sphinx-theme/blob/main/src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-button-field.html search button field in our components, expecially some custom class names. 
>
> so to use ansys sphinx theme, search we might use search-button-field

Agreed!